src/libraries/System.Net.Quic/src/Resources/Strings.resx

```diff
@@ -150,5 +150,14 @@
   <data name="net_quic_writing_notallowed" xml:space="preserve">
     <value>Writing is not allowed on stream.</value>
   </data>
+  <data name="net_quic_ssl_optiopn" xml:space="preserve">
```

Comment:
Where is this used?  It's also mispelled :)

Reply:
that is pollution from #55877. (where the spelling was corrected)